components/server/ee/src/workspace/workspace-factory.ts

```diff
@@ -177,6 +178,10 @@ export class WorkspaceFactoryEE extends WorkspaceFactory {
                 statusVersion: 0,
             });
 
+            if (pws) {
+                increasePrebuildsStartedCounter();
```

Comment:
This is also affected by the multi-replica problem. So depending on which solution you go for, it needs to be done here too.